settings.gradle.kts

```diff
@@ -114,7 +114,7 @@ dependencyResolutionManagement {
     // runtime.
     create("libs") {
       // The HAPI API version to use, this need to match the tag set on gitRepositories above
-      version("hapi-version", "0.36.1")
+      version("hapi-version", "0.37.0")
```

Comment:
This should be a SNAPSHOT artifact. We release non snapshot artifacts at end of release cycle